flake.nix

```
@@ -1,50 +1,35 @@
 {
   description = "An attempt to better support Minecraft-related content for the Nix ecosystem";
 
-  inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-    flake-utils.url = "github:numtide/flake-utils";
-  };
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
 
-  outputs =
-    { self
-    , nixpkgs
-    , flake-utils
-    , ...
-    }@inputs:
+  outputs = { self, nixpkgs, ... }@inputs:
     let
-      packages = pkgs:
+      supportedSystems = [
+        "aarch64-linux"
+        "aarch64-darwin"
```

Nitpick: 
```suggestion
        "aarch64-darwin"
        "aarch64-linux"
```
Sorted alphabetically.

---

Ah i was actually attempting to alphabetically sorting them. But i have seem to have failed my ABCs here :sweat_smile: